resources/assets/coffee/forum/topic-ajax.coffee

```diff
@@ -40,6 +40,14 @@ $(document).on 'ajax:success', '.delete-post-link', (_event, data) ->
       $(document).trigger('osu:page:change')
 
 
+$(document).on 'ajax:before', '.reply-post-link', (e, data) ->
+  if window.isDoublePost
+    url = location.href
+    history.replaceState(null, null, url)
+    location.href = "#doublepost-box"
```

Comment:
must have overseen that, hmm. Any suggestions how to load / find the necessary post if it's not loaded yet?